plugins/editorplaceholder/plugin.js

```diff
@@ -21,13 +21,12 @@
 				return;
 			}
 
-			bindPlaceholderEvents( editor, [
-				'contentDom',
-				'focus',
-				'blur',
-				'change'
-			] );
+			bindPlaceholderEvent( editor, 'contentDom' );
+			bindPlaceholderEvent( editor, 'focus' );
+			bindPlaceholderEvent( editor, 'blur' );
```

Comment:
These events are not debounced on purpose, as debounce logic on focus/blur creates a visible placeholder delay. In probably 99% of cases we could simply skip toggling editor placeholder on the `change` event as, during normal editor usage, the editor placeholder is not visible until you blur the editor. However, a `change` event is required for API manipulation like `editor.insertText`.